docs/setup/config.md

```diff
@@ -275,6 +275,10 @@ For example when running Flink on YARN on an environment with a restrictive fire
 
 - `recovery.job.delay`: (Default 'akka.ask.timeout') Defines the delay before persisted jobs are recovered in case of a recovery situation. 
 
+## Environment
+
+- `env.log.dir`: (Defaults to the `log` directory under Flink's home) Defines the directory where the Flink logs are saved.
```

Comment:
I did some research but checking for a valid path across all possible operative systems and filesystems would be prohibitive, I guess we'll have to settle for the documentation at this time. :smiley: Thanks for pointing it out!